### ISSUE TITLE
Django 2.1: AttributeError: module 'django.db.models' has no attribute 'permalink'

### ISSUE THREAD

**Issue by andrewpwade** [cyface/django-termsandconditions#58]
```
  File ".../lib/python3.6/site-packages/termsandconditions/models.py", line 63, in TermsAndConditions
    @models.permalink
AttributeError: module 'django.db.models' has no attribute 'permalink'
```

Django 2.1 has [removed](https://docs.djangoproject.com/en/2.1/releases/2.1/#features-removed-in-2-1) the permalink() decorator.

I think a fix would be something like:

```python
from django.urls import reverse

...

    def get_absolute_url(self):
        return reverse(
            'tc_view_specific_version_page', 
            args=[self.slug, self.version_number])  # pylint: disable=E1101
```
Thanks for this app!

**Comment by cyface** [comment]
Thanks for the report! Branch in progress to address this. 👍

**Comment by cyface** [comment]
Fixed in v1.2.12! Thanks again!

**Comment by anordinho** [comment]
it didint work for me so what can i do
